go

```diff
@@ -59,7 +59,7 @@ def init
 end
 
 def update_gems
-  exec_cmd 'bundle update'
+  exec_cmd 'bundle install'
```

Comment:
Yeah, that's what we're trying to avoid. We did the same thing on 18F/18f.gsa.gov after an unexpected upgrade started throwing problems on the server.